### ISSUE TITLE
Web page is not being navigated sequentially with the Tab key at Editor screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#825]
**Describe the bug**
Web page is not being navigated sequentially with the Tab key at Editor screen. The sequence of focus is unordered.

**To Reproduce**
1. Go to Editor screen and select any page for a site and edit it. 
2. Use Tab key to focus on different components of the screen.
3. Focus is not moving in sequence and is not going to "Layout" and "Style" Tabs. Also, focus is not going to Actions, View, Publishing, etc.

**Expected behavior**
Web page should be navigated sequentially with the Tab key.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by rgarg24** [comment]
Duplicate issue. Bug already raised #794.
